src/Mgate/PersonneBundle/Resources/views/Membre/impayes.html.twig

```diff
@@ -45,12 +46,17 @@
                     <a href="{{ path('MgatePersonne_membre_voir', {'id': membre.id}) }}">{{ membre.personne ? membre.personne.nom }}</a>
                 </td>
                 <td>{{ membre.personne ? membre.personne.email }}</td>
-                <td style="text-align: center;"><h4><span
-                                class="label label-default">{{ membre.promotion ? "P" ~ membre.promotion }}</span>
+                <td class="text-center">
+                    <span class="label label-default">{{ membre.promotion ? "P" ~ membre.promotion }}</span>
                 </td>
                 <td>{{ membre.personne ? membre.personne.mobile }}</td>
-                <td>{{ membre.personne ? membre.personne.adresse }} {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}</td>
+                <td>
+                    {{ membre.personne ? membre.personne.adresse }}<br/>
+                    {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}
+                </td>
             </tr>
+        {% else %}
+            <li>{{ 'member.empty_unpaid_list_description'|trans({}, 'person') }}</li>
```

Comment:
`<tr><td colspan="6">Pas d'impayés</td></tr>` instead of `<li>`.

Not sure it is even necessary because datatables will display an empty table message.